test/hex_web/stats_job_test.exs

```diff
@@ -24,16 +24,18 @@ defmodule HexWeb.StatsJobTest do
 
   test "counts all downloads" do
     buckets = Application.get_env(:hex_web, :logs_buckets)
+
     {bucket, region} =
       if buckets do
         Application.put_env(:hex_web, :store, HexWeb.Store.S3)
         [[bucket, region]] = buckets
         {bucket, region}
       else
-        buckets = [[nil, nil]]
         {nil, nil}
       end
 
+    buckets = [[nil, nil]]
```

Comment:
Isn't that overriding the buckets value from line 31? Maybe you meant `buckets = buckets || [[nil, nil]]`?